deferred-leveldown.js

```diff
@@ -41,29 +42,40 @@ DeferredLevelDOWN.prototype._close = function (callback) {
   })
 }
 
-function open (obj) {
+function open (self) {
   deferrables.concat('iterator').forEach(function (m) {
-    obj['_' + m] = function () {
+    self['_' + m] = function () {
       return this._db[m].apply(this._db, arguments)
     }
   })
+  if (self._db.approximateSize) {
+    self.approximateSize = function () {
+      return this._db.approximateSize.apply(this._db, arguments)
+    }
+  }
 }
 
-function closed (obj) {
+function closed (self, db) {
```

Comment:
The `db` argument is unused?

Reply:
Good catch. I originally added it but later changed to `self._db` inside the function.